src/components/Infobox.tsx

```diff
@@ -6,7 +6,8 @@ import OrangeGhost from '../../static/svg/orange-ghost.svg'
 export type InfoboxType = 'warning'|'sidenote'
 
 interface InfoboxProps extends PropsWithChildren {
-  type: InfoboxType
+  type: InfoboxType,
+  customTitle: string
```

Comment:
```suggestion
  title: string
```

I prefer `title` over `customTitle`. Up to you !